doc/externalGrading.md

```diff
@@ -167,11 +167,14 @@ Your grading process must write its results to `/grade/results/results.json`. Th
 
```

Comment:
Perhaps link out to the `python-grader` help page and source as an implementation of the external grader spec?